extensions/publisher-cves/src/components/ContentList.vue

```diff
@@ -0,0 +1,77 @@
+<script setup lang="ts">
+import { usePackagesStore } from "../stores/packages";
+import { useContentStore } from "../stores/content";
+import StatusMessage from "./ui/StatusMessage.vue";
+import ContentCard from "./ContentCard.vue";
+
+const packagesStore = usePackagesStore();
+const contentStore = useContentStore();
+
+// Fetch packages in batches to avoid overwhelming the server
+async function fetchPackagesInBatches(batchSize = 3) {
```

Comment:
I debated firing off all of the fetches to grab Connect content packages, but figured batching it didn't cause many problems. It starts with the top content item and slowly loads in which is nice from the user perspective.